transforms/images/apply-flatfield-plugin/run-plugin.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+version=$(<VERSION)
+datapath=$(readlink --canonicalize ./data)
+echo ${datapath}
+
+# Inputs
+imgDir="/data/images"
+imgPattern="p{p:d+}_x{x:d+}_y{y:d+}_wx{r:d+}_wy{z:d+}_c{c:d+}.ome.tif"
+ffDir="/data/estimation"
+brightPattern="p{p:d+}_x\\(01-24\\)_y\\(01-16\\)_wx\\(1-3\\)_wy\\(1-3\\)_c{c:d+}_flatfield.ome.tif"
+darkPattern="p{p:d+}_x\\(01-24\\)_y\(01-16\\)_wx\\(1-3\\)_wy\\(1-3\\)_c{c:d+}_darkfield.ome.tif"
+# photoPattern=""
+
+# Output paths
+outDir=/data/outputs
+
+FILE_EXT=".ome.zarr"
```

Comment:
Why this FILE_EXT is there?

Reply:
This is the env-variable used to have the output images be ome-zarr. It only applies when using the run-plugin script